test/e2e/Makefile

```diff
@@ -28,7 +28,7 @@ export GOPROXY
 REPO_ROOT := $(shell git rev-parse --show-toplevel)
 
 help:  ## Display this help
-	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
```

Comment:
Was this change intentional? What does it do?

Reply:
moved to #3651